.azure-pipelines/ci.yml

```
@@ -1,4 +1,4 @@
-trigger: ['main', '3.13', '3.12', '3.11', '3.10', '3.9', '3.8']
+trigger: ['main', '3.13', '3.12', '3.11', '3.10', '3.9']
```

Does Azure Pipelines work with wildcards, like GitHub Actions?

https://github.com/python/cpython/blob/1b49c8c71b90bfa97df5633e2bbf51d4a6e22a57/.github/workflows/build.yml#L8

```suggestion
trigger: ['main', '3.*']
```

---

It should: https://learn.microsoft.com/en-us/azure/devops/pipelines/customize-pipeline?view=azure-devops#customize-ci-triggers.